src/base-implementation.ts

```diff
@@ -0,0 +1,103 @@
+import { Wallet } from '@harmony-js/account'
```

Comment:
I would call this file `base-token.ts` and remove the abstract base token created

Reply:
Copy!

Reply:
Copy!